adafruit_mpu6050.py

```diff
@@ -74,13 +75,36 @@
 STANDARD_GRAVITY = 9.80665
 
 
+class Clksel:  # pylint: disable=too-few-public-methods
```

Comment:
Can you give this a more "descriptive" name?  A lot of people will interface with this class, so an intuitive name would help.  Perhaps `ClockSource`?

Reply:
Okay, I can change it to whatever the maintainers think is appropriate.  

My thinking was the register map refers to it as the  "clksel" register.  So anyone who looks into documentation won't find a 'clock source' register.  I was hoping hiding the 'clksel' register behind the 'clock source' method and 'clksel' class would work.  But I'm open to whatever you think is appropriate.